packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Viewport positioner",
+    "description": "A viewport positioner component's schema definition.",
+    "type": "object",
+    "id": "@microsoft/fast-components-react-base/viewport-positioner",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "horizontalPositioningMode": {
+            "title": "Horizontal positioning mode",
+            "type": "string",
+            "default": "uncontrolled",
+            "enum": [
+                "uncontrolled",
+                "flipOutward",
+                "flipInward"
+            ]
+        },
+        "defaultHorizontalPosition": {
+            "title": "Default horizontal position",
+            "type": "string",
+            "default": "uncontrolled",
+            "enum": [
+                "left",
+                "centerLeft",
+                "centerRight",
+                "right"
```

Comment:
you should add "uncontrolled" to the enum

Reply:
yep